drivers/SmartThings/zigbee-switch/src/configurations.lua

```diff
@@ -60,8 +63,29 @@ local devices = {
   }
 }
 
+
 local configurations = {}
 
+configurations.power_reconfig_wrapper = function(orig_function)
+  local new_init = function(driver, device)
+    local config_version = device:get_field(CONFIGURATION_VERSION_KEY)
+    if config_version == nil or config_version < driver.current_config_version then
```

Comment:
I think we should also be using `Device:add_configured_attribute` to ensure if configure is called (I think some subdrivers do this from refresh handler) that the new configuration isn't overwritten.

Reply:
Done